json-schemas/src/app-stats.json

```diff
@@ -9,632 +9,786 @@
     },
     "unit": {
       "type": "string",
-      "description": "unit of time for these stats from the intervalId forwards",
+      "description": "Unit of time for these stats from the intervalId forwards.",
       "pattern": "^minute$"
     },
     "schema": {
       "type": "string",
-      "description": "URI to this schema"
+      "description": "URI to this schema."
     },
     "entries": {
       "type": "object",
-      "description": "stats entries",
+      "description": "Stats entries.",
       "properties": {
         "messages.all.all.count": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total message count."
         },
         "messages.all.all.data": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total message size."
         },
         "messages.all.all.uncompressedData": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total uncompressed message size."
```

Comment:
We might want to put a bit more info in here about what exactly 'uncompressed message size' means, currently the descriptions field isn't really adding much to the metric name - cf https://github.com/ably/realtime/issues/2462